x-pack/plugins/security_solution/common/endpoint/models/policy_config_helpers.test.ts

```diff
@@ -5,6 +5,7 @@
  * 2.0.
  */
 
+import { createDefaultPolicy } from '../../../server/fleet_integration/handlers/create_default_policy';
```

Comment:
is this passing CI?  I did not think you could import from `server/` from inside of `/common` (possible circular dependencies)